-- Found rosidl_default_generators: 1.0.0 (/home/hyunjong/ros2_foxy/install/rosidl_default_generators/share/rosidl_default_generators/cmake)
-- Found ament_cmake_core: 0.9.6 (/home/hyunjong/ros2_foxy/install/ament_cmake_core/share/ament_cmake_core/cmake)
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.2", minimum required is "3") 
-- Using PYTHON_EXECUTABLE: /usr/bin/python3
-- Using all available rosidl_typesupport_c: rosidl_typesupport_introspection_c;rosidl_typesupport_fastrtps_c
-- Found rosidl_adapter: 1.0.1 (/home/hyunjong/ros2_foxy/install/rosidl_adapter/share/rosidl_adapter/cmake)
-- Using all available rosidl_typesupport_cpp: rosidl_typesupport_introspection_cpp;rosidl_typesupport_fastrtps_cpp
-- Found PythonInterp: /usr/bin/python3 (found version "3.8.2") 
CMake Error at /home/hyunjong/ros2_foxy/install/rosidl_adapter/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
  execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
  tutorial_interfaces --arguments-file
  /home/hyunjong/ros2_ws/build/tutorial_interfaces/rosidl_adapter__arguments__tutorial_interfaces.json
  --output-dir
  /home/hyunjong/ros2_ws/build/tutorial_interfaces/rosidl_adapter/tutorial_interfaces
  --output-file
  /home/hyunjong/ros2_ws/build/tutorial_interfaces/rosidl_adapter/tutorial_interfaces.idls)
  returned error code 1:

  Error with: tutorial_interfaces AddThreeInts_Request int64	a

  Traceback (most recent call last):

    File "/usr/lib/python3.8/runpy.py", line 193, in _run_module_as_main
      return _run_code(code, main_globals, None,
    File "/usr/lib/python3.8/runpy.py", line 86, in _run_code
      exec(code, run_globals)
    File "/home/hyunjong/ros2_foxy/install/rosidl_adapter/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
      sys.exit(main())
    File "/home/hyunjong/ros2_foxy/install/rosidl_adapter/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
      abs_idl_file = convert_to_idl(
    File "/home/hyunjong/ros2_foxy/install/rosidl_adapter/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
      return convert_srv_to_idl(
    File "/home/hyunjong/ros2_foxy/install/rosidl_adapter/lib/python3.8/site-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
      srv = parse_service_string(package_name, input_file.stem, content)
    File "/home/hyunjong/ros2_foxy/install/rosidl_adapter/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 836, in parse_service_string
      request_message = parse_message_string(
    File "/home/hyunjong/ros2_foxy/install/rosidl_adapter/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 497, in parse_message_string
      raise InvalidFieldDefinition(line)

  rosidl_adapter.parser.InvalidFieldDefinition: int64	a

Call Stack (most recent call first):
  /home/hyunjong/ros2_foxy/install/rosidl_cmake/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
  CMakeLists.txt:6 (rosidl_generate_interfaces)


-- Configuring incomplete, errors occurred!
See also "/home/hyunjong/ros2_ws/build/tutorial_interfaces/CMakeFiles/CMakeOutput.log".
